.gitignore

```diff
@@ -111,4 +111,3 @@ dist
 # Stores VSCode versions used for testing VSCode extensions
 .vscode-test
 
```

Comment:
jwt.key
jwt.key.pub

Reply:
not technically necessary since the config folder is ignored, but good idea in case they somehow aren't